[The One Ring 2e] 2.2.1 - next try to fix the css roll template bug

bhuesemann opened the pull request:
## Changes / Comments

### CSS fixes

- removed font imports and font-face declarations
- added two missing .sheet prefixes in roll-template
- added two missing .charsheet prefixes in the pc css
- fixed a typo in the roll-template css

## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [x] Does the pull request title have the sheet name(s)? Include each sheet name.
- [x] Is this a bug fix?
- [ ] Does this add functional enhancements (new features or extending existing features) ?
- [ ] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [ ] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- [ ] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

roll20deploy commented:
[Character Sheet Info](https://app.roll20.net/managesheets/The%20One%20Ring%202e/roll20-character-sheets) *Roll20 Internal Use only.*

bhuesemann commented:
@nmbradley Hi Nic, with this PR I've also removed the fonts imports completely and fixed two other minor issues in the css of the roll-template. Hopefully this will make this sheet usable.

bhuesemann commented:
replaced by version 2.3.0
